realm/realm-library/src/androidTest/java/io/realm/ObjectChangeSetTests.java

```diff
@@ -291,6 +290,48 @@ public void changeAllFields() {
 
     @Test
     @RunTestInLooperThread(before = PopulateOneAllTypes.class)
+    public void  changeDifferentFieldOneAfterAnother() {
```

Comment:
Perhaps add a comment about why this is relevant or link to bug report. Also too much whitespace